Running tests fails

Running the tests with tox fails, because it also checks the documentation and plots figures while running the tests.
The tests work when no documentation is built. However, once you build the documentation, the tests fail.

This is the error I get when running `tox -e test`:

________________________________ ERROR collecting docs/_build/html/calc_extinction-3.py _________________________________
../../docs/_build/html/calc_extinction-3.py:24: in <module>
    fig, ax = plt.subplots()
E   NameError: name 'plt' is not defined
________________________________ ERROR collecting docs/_build/html/calc_extinction-4.py _________________________________
../../docs/_build/html/calc_extinction-4.py:29: in <module>
    fig, ax = plt.subplots()
E   NameError: name 'plt' is not defined
=================================================== warnings summary ====================================================
/Users/mdecleir/opt/anaconda3/envs/astroconda/lib/python3.6/importlib/_bootstrap.py:219: 25 tests with warnings
  /Users/mdecleir/opt/anaconda3/envs/astroconda/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
    return f(*args, **kwds)

/Users/mdecleir/measure_extinction/.tox/test/lib/python3.6/site-packages/stsynphot/spectrum.py:618
  /Users/mdecleir/measure_extinction/.tox/test/lib/python3.6/site-packages/stsynphot/spectrum.py:618: AstropyUserWarning: Failed to load Vega spectrum from /grp/hst/cdbs//calspec/alpha_lyr_stis_009.fits; Functionality involving Vega will be cripped: FileNotFoundError(2, 'No such file or directory')
    AstropyUserWarning)

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-1.py:26
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-1.py:26: DeprecationWarning: invalid escape sequence \l
    ax.set_xlabel('$\lambda$ [$\mu m$]')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-1.py:27
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-1.py:27: DeprecationWarning: invalid escape sequence \l
    ax.set_ylabel('$F(\lambda)$ [$ergs\ cm^{-2}\ s\ \AA$]')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-2.py:28
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-2.py:28: DeprecationWarning: invalid escape sequence \l
    ax.set_xlabel('$\lambda$ [$\mu m$]')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-2.py:29
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-2.py:29: DeprecationWarning: invalid escape sequence \l
    ax.set_ylabel('$E(\lambda - V)$ [mag]')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-3.py:32
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-3.py:32: DeprecationWarning: invalid escape sequence \l
    ax.set_xlabel('$\lambda$ [$\mu m$]')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-3.py:33
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-3.py:33: DeprecationWarning: invalid escape sequence \l
    ax.set_ylabel('$E(\lambda - V)/E(B-V)$')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-4.py:38
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-4.py:38: DeprecationWarning: invalid escape sequence \l
    ax.set_xlabel('$\lambda$ [$\mu m$]')

/Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-4.py:39
  /Users/mdecleir/measure_extinction/docs/_build/html/calc_extinction-4.py:39: DeprecationWarning: invalid escape sequence \l
    ax.set_ylabel('$A(\lambda)/A(V)$')

-- Docs: https://docs.pytest.org/en/latest/warnings.html
================================================ short test summary info ================================================
ERROR ../../docs/_build/html/calc_extinction-3.py - NameError: name 'plt' is not defined
ERROR ../../docs/_build/html/calc_extinction-4.py - NameError: name 'plt' is not defined
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
====================================== 34 warnings, 2 errors in 120.82s (0:02:00) =======================================
ERROR: InvocationError for command /Users/mdecleir/measure_extinction/.tox/test/bin/pytest --pyargs measure_extinction /Users/mdecleir/measure_extinction/docs (exited with code 2)
________________________________________________________ summary ________________________________________________________
ERROR:   test: commands failed

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -65,7 +65,7 @@ fixes, code, or documentation (no git or GitHub experience necessary):
 
 * `Developer Documentation <https://docs.astropy.org/en/stable/#developer-documentation>`_
 
-For the complete list of contributors please see the `dust_extinction
+For the complete list of contributors please see the `measure_extinction
 contributors page on Github
 <https://github.com/karllark/measure_extinction/graphs/contributors>`_.
 

--- a/measure_extinction/conftest.py
+++ b/measure_extinction/conftest.py
@@ -37,7 +37,7 @@ def pytest_configure(config):
 
 # Uncomment the last two lines in this block to treat all DeprecationWarnings as
 # exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
-# as follow (although default should work for most cases).
+# as follows (although default should work for most cases).
 # To ignore some packages that produce deprecation warnings on import
 # (in addition to 'compiler', 'scipy', 'pygments', 'ipykernel', and
 # 'setuptools'), add:

--- a/setup.cfg
+++ b/setup.cfg
@@ -40,6 +40,9 @@ astropy_header = true
 doctest_plus = enabled
 text_file_format = rst
 addopts = --doctest-rst
+norecursedirs =
+    measure_extinction/docs/_build/*
+    measure_extinction/docs/api/*
 
 [coverage:run]
 omit =

